delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/QueryBuilder.scala

```diff
@@ -137,14 +137,20 @@ final case class QueryBuilder private[client] (private val query: JsonObject) {
     }
   }
 
-  private def term[A: Encoder](k: String, value: A): JsonObject              =
+  private def term[A: Encoder](k: String, value: A): JsonObject             =
     JsonObject("term" -> Json.obj(k -> value.asJson))
 
-  private def terms[A: Encoder](k: String, values: Iterable[A]): JsonObject  =
+  private def terms[A: Encoder](k: String, values: Iterable[A]): JsonObject =
     JsonObject("terms" -> Json.obj(k -> values.asJson))
 
-  private def matchPhrasePrefix[A: Encoder](k: String, value: A): JsonObject =
-    JsonObject("match_phrase_prefix" -> Json.obj(k -> Json.obj("query" -> value.asJson)))
+  private def multiMatch[A: Encoder](value: A): JsonObject                  =
+    JsonObject(
+      "multi_match" -> Json.obj(
+        "query"  -> value.asJson,
+        "type"   -> Json.fromString("phrase_prefix"),
+        "fields" -> json"""[ "@id.fulltext^10", "*", "*.fulltext", "_tags", "_original_source", "_uuid" ]"""
```

Comment:
description is missing here

Reply:
From what I tried and what I understand it will be matched by `*`. Ideally I would have liked to have just `"@id.fulltext^10", "*", "*.fulltext"`. The issue with `"_tags", "_original_source", "_uuid"` is that they were already `text` fields; but they are not caught by the `*` (because they start with `_`), and they cannot be caught by `*.fulltext` because you cannot add a multi-field which is a duplicate of the main type

Reply:
There is a test now that matches on the description that shows it works

Reply:
I also removed the multiplier on `@id`, because I don't think we need it now that IRIs use the path_hierarchy search analyzer